site/sakai-message-bundle-manager-tool/src/main/java/org/sakaiproject/mbm/tool/MainController.java

```diff
@@ -111,6 +140,33 @@ public String showModified(Model model) {
 	    return "modified";
 	}
 	
+    @RequestMapping(value = "/modified", params = {"dtf=csv","dti=modified"}, method = RequestMethod.GET, produces = "text/csv")
```

Comment:
Yeah it appears there are tabs in the original file but my IDE is adding spaces instead of tabs.

It looks fine in the IDE but I am not going to remove all the tabs at this point.